src/api/draftSpecs.ts

```diff
@@ -169,7 +166,7 @@ export const deleteDraftSpecsByCatalogName = async (
         // In case we get an absolutely massive amount of catalogs to delete,
         // we don't want to spam supabase
         const limiter = pLimit(3);
-        const promises: Array<Promise<PostgrestResponse<any>>> = [];
+        const promises: Array<Promise<PostgrestSingleResponse<any>>> = [];
```

Comment:
The typing from the library changed. In the code base it looks like the will further change this typing so there is only a single kind of response.